en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -45,8 +45,21 @@ The actions you can perform in a throughput graph are as follows:
      
 - **View data for a custom time range:**
   You can expand the graph by clicking and dragging the pointer over the period you want to drill down. The default graph displays data for a custom time range where the data points are at a lower granularity.
+  
+- **View details per request** 
 
- 
+    Once you view data for a custom time range, each request that was sent during the selected time range is displayed in a pane above the low-code diagram as shown below.
+    
+    ![Trace Requests](../assets/img/observability/view-details-per-request.png){.cInlineImage-full}
+
+    This pane displays the latency, the time at which the service received the request, and the status.
+    
+    When you click on a specific request, the low-code diagram displays the following:
+   
+    - If an error has occurred, the error code is displayed near the relevant API call. In this example, the `500` error code has occurred when the request was processed by an API call.
+    - The path (control flow) in which the service executed the request is highlighted in green. In this example, the highlighted path indicates that the selected request was processed on the `else` path of the first `if` statement, and on the `then` path of the second `if` statement.
+    - The execution time is displayed per `if`, `while`, and `foreach` body. In this example, the execution time displayed for the `if` body is `603 μs`.
```

Comment:
Should we say:
"In this example, the execution time taken for the `if` body is `603 μs`."

Reply:
Done